src/backend/utils/adt/agtype.c

```diff
@@ -151,25 +159,25 @@ static bool is_array_path(agtype_value *agtv);
 /* graph entity retrieval */
 static Datum get_vertex(const char *graph, const char *vertex_label,
                         int64 graphid);
-static char *get_label_name(const char *graph_name, int64 graph_id);
+static char *get_label_name(const char *graph_name, int64 label_id);
```

Comment:
The second parameter is a vertex\edge ID (as seen in where the function is called from). Not a label ID.

Ideally, the type should be `graphid` instead of `int64`.

Reply:
I think that, overall, we have an issue with the name `graphid ` as a type due to names of fields and variables like `graph_id` or `graph_oid`. Additionally, `graphid ` is just an `int64`. So, I don't even know if we need to have a type for it and then a field named like it. Still, I'm not sure if we should attempt to fix this in this PR.

Reply:
`graphid` is referred to as bit shifted label_id with entry_id. So, I believe it is inteneded to be used as vertex and edge ID only.
I agree that we may not need to change it in this PR.